.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   ci:
     if: "!contains(github.event.commits[0].message, '[ci skip]')"
-    name: 'Ruby ${{ matrix.ruby_version }}'
+    name: 'Ruby ${{ matrix.ruby_version }} '
```

Comment:
Would you mind removing the extra whitespace here inside the quote here?

Reply:
No Problem, fixed!